generators/entity-server/templates/src/main/java/package/service/impl/_EntityServiceImpl.java

```diff
@@ -36,8 +36,10 @@
 import <%=packageName%>.service.mapper.<%= entityClass %>Mapper;<% } %>
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+<%_ if (pagination !== 'no') { _%>
```

Comment:
I need to revert this, as this makes the tests fail, when there is a relationship (hava a look at the `findAllWithEagerRelationships` method)

Reply:
oh I'll try with a  `|| fieldsContainOwnerManyToMany === true`